Lombiq.Tests.UI/Extensions/AssemblyResourceExtensions.cs

```diff
@@ -1,39 +1,17 @@
 using SixLabors.ImageSharp;
 using System.Reflection;
 
-using DrawingBitmap = System.Drawing.Bitmap;
-using DrawingImage = System.Drawing.Image;
-
 namespace Lombiq.Tests.UI.Extensions;
 
 public static class AssemblyResourceExtensions
 {
     /// <summary>
-    /// Loads resource specified by name from the given assembly.
+    /// Loads resource specified by <paramref name="name"/> from the given <paramref name="assembly"/>.
     /// </summary>
-    /// <param name="name">Resource name.</param>
-    /// <returns><see cref="Image"/> instance.</returns>
     public static Image GetResourceImageSharpImage(this Assembly assembly, string name)
     {
         using var resourceStream = assembly.GetManifestResourceStream(name);
 
         return Image.Load(resourceStream);
     }
-
-    /// <summary>
-    /// Loads resource specified by name from the given assembly.
-    /// </summary>
-    /// <param name="name">Resource name.</param>
-    /// <returns><see cref="DrawingBitmap"/> instance.</returns>
-    public static DrawingBitmap TryGetResourceBitmap(this Assembly assembly, string name)
```

Comment:
If I remove baseline image, eg.: [TestCaseUITestContextExtensions_TestChartJsBarChartAsync_By_TagName_-canvas.png](https://github.com/Lombiq/Orchard-Chart.js/blob/issue/OSOE-403/Lombiq.ChartJs.Tests.UI/Extensions/TestCaseUITestContextExtensions_TestChartJsBarChartAsync_By_TagName_-canvas.png)
I get this when running `Lombiq.OSOCE.Tests.UI.Tests.ModuleTests.BehaviorChartJsTests.RecipeDataShouldBeDisplayedCorrectly`.
See: https://github.com/Lombiq/UI-Testing-Toolbox/blob/7f3fa213064888439e7b391f88f860f30f4975bc/Lombiq.Tests.UI/Extensions/VisualVerificationUITestContextExtensions.cs#L308

```
  Message: 
System.ArgumentNullException : Parameter "stream" must be not null. (Parameter 'stream')

  Stack Trace: 
ThrowHelper.ThrowArgumentNullException(String name, String message)
ThrowHelper.ThrowArgumentNullExceptionForNotNull(String name)
Image.WithSeekableStream[T](Configuration configuration, Stream stream, Func`2 action)
Image.Load(Configuration configuration, Stream stream, IImageFormat& format)
Image.Load(Configuration configuration, Stream stream)
Image.Load(Stream stream)
AssemblyResourceExtensions.GetResourceImageSharpImage(Assembly assembly, String name) line 15
VisualVerificationUITestContextExtensions.AssertVisualVerificationApproved(UITestContext context, IWebElement element, Action`2 comparator, Nullable`1 regionOfInterest, Action`1 configurator) line 305
VisualVerificationUITestContextExtensions.AssertVisualVerificationApproved(UITestContext context, By elementSelector, Action`2 comparator, Nullable`1 regionOfInterest, Action`1 configurator) line 245
VisualVerificationUITestContextExtensions.AssertVisualVerificationApproved(UITestContext context, By elementSelector, Double pixelErrorPercentageThreshold, Nullable`1 regionOfInterest, Action`1 configurator) line 77
TestCaseUITestContextExtensions.TestChartJsChart(UITestContext context, String logHeader, Double pixelErrorPercentageThreshold) line 51
TestCaseUITestContextExtensions.TestChartJsBarChartAsync(UITestContext context) line 25
TestCaseUITestContextExtensions.TestChartJsSampleBehaviorAsync(UITestContext context) line 17
UITestExecutionSession`1.ExecuteAsync(Int32 retryCount, String dumpRootPath) line 110
UITestExecutionSession`1.ExecuteAsync(Int32 retryCount, String dumpRootPath) line 134
UITestExecutionSession`1.ExecuteAsync(Int32 retryCount, String dumpRootPath) line 151
UITestExecutor.ExecuteOrchardCoreTestInnerAsync[TEntryPoint](UITestManifest testManifest, OrchardCoreUITestExecutorConfiguration configuration, String dumpRootPath) line 83
UITestExecutor.ExecuteOrchardCoreTestInnerAsync[TEntryPoint](UITestManifest testManifest, OrchardCoreUITestExecutorConfiguration configuration, String dumpRootPath) line 83
OrchardCoreUITestBase`1.ExecuteTestAsync(Func`2 testAsync, Browser browser, Func`2 setupOperation, Func`2 changeConfigurationAsync) line 253
--- End of stack trace from previous location ---
```

Reply:
How do you remove the baseline image? If I delete it from the project, I get a build error:
>   Error reading resource 'Lombiq.ChartJs.Tests.UI.Extensions.TestCaseUITestContextExtensions_TestChartJsBarChartAsync_By_TagName_-canvas.png' -- 'Could not find file 'C:\repositories\orchard-core\Open-Source-Orchard-Core-Extensions3\src\Modules\Lombiq.ChartJs\Lombiq.ChartJs.Tests.UI\Extensions\TestCaseUITestContextExtensions_TestChartJsBarChartAsync_By_TagName_-canvas.png'.'

It seems to me that it should not be removed.

Reply:
I'm using VS and deleted it from the csproj.

The `TryGetResourceBitmap` is required because:

1. When creating a new visual verification test, and it returns `null` on the first run, the captured image is copied next to the source file from where the  `AssertVisualVerificationApproved` got called, and it will be used as a baseline image. 
2. When something changed on the UI and you need a new baseline image, you can simply delete it and the process is the same as described under 1.

Reply:
See: https://github.com/Lombiq/UI-Testing-Toolbox/blob/7f3fa213064888439e7b391f88f860f30f4975bc/Lombiq.Tests.UI/Extensions/VisualVerificationUITestContextExtensions.cs#L632